vpn/README.md

```diff
@@ -75,24 +70,18 @@ copy paste the output into Outline Manager (formatted as JSON).
 
 Potentially, we can get this in Terraform, find a draft in the experimental PR [#33](https://github.com/dbl-works/terraform/pull/33).
 
-
```

Comment:
we usually keep 3 new lines between headlines intentionally, to make it easier to quickly scan the readme when scrolling through it.